src/wp-admin/includes/class-wp-plugins-list-table.php

```diff
@@ -759,6 +759,54 @@ public function single_row( $item ) {
 		$compatible_php = is_php_version_compatible( $requires_php );
 		$compatible_wp  = is_wp_version_compatible( $requires_wp );
 
+		if ( ! function_exists( 'plugins_api' ) ) {
+			require_once ABSPATH . 'wp-admin/includes/plugin-install.php';
+		}
+
+		$plugin_information = plugins_api(
+			'plugin_information',
+			array(
+				'slug'   => $plugin_slug,
+				'fields' => array(
+					'sections' => false,
+					'rating'   => false,
+					'downloaded' => false,
+					'last_updated' => false,
+					'banners' => false,
+					'icons' => false,
+					'active_installs' => false,
+					'short_description' => false,
+					'tags' => false,
+					'compatibility' => false,
+					'contributors' => false,
+					'ratings' => false,
+					'screenshots' => false,
+					'support_url' => false,
+					'added' => false,
+					'homepage' => false,
+					'download_link' => false,
+					'upgrade_notice' => false,
+					'versions' => false,
+					'business_model' => false,
+					'repository_url' => false,
+					'commercial_support_url' => false,
+					'donate_link' => false,
+					'preview_link' => false,
+				),
+			)
+		);
+
+		$tested_wp         = null;
+		$tested_compatible = true;
+
+		if ( ! is_wp_error( $plugin_information ) && isset( $plugin_information->tested ) ) {
+			$tested_wp = $plugin_information->tested;
+		}
+
+		if ( null !== $tested_wp ) {
+			$tested_compatible = is_tested_wp_version_compatible( $tested_wp );
+		}
```

Comment:
This can be simplified because checking for `null` is the same as checking `isset()`:

```suggestion
		if ( ! is_wp_error( $plugin_information ) && isset( $plugin_information->tested ) ) {
			$tested_compatible = is_tested_wp_version_compatible( $plugin_information->tested );
		}
```